src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php

```diff
@@ -65,6 +65,23 @@ private function hasDefaultValue(\ReflectionParameter $parameter)
     }
 
     /**
+     * Returns if the argument is allowed to be null but is still mandatory.
+     *
+     * @param \ReflectionParameter $parameter
+     *
+     * @return bool
+     */
+    private function isNullable(\ReflectionParameter $parameter)
+    {
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
Not necessarilly, for example hhvm can support some php7 features and set this constant to php5 (see [this comment](https://github.com/symfony/symfony/pull/19191#discussion_r69262173)) and as we don't know how they are gonna support php7.1 features, i think it is safer to use `method_exists`.